getting-started/repl/intergration.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Integration
+page_title: Telerik REPL for Blazor Integration
+description: Telerik REPL for Blazor integration in other Telerik Resources, Visual Studio and Visual Studio Code
+slug: blazor-repl-integration
+tags: telerik,blazor,repl,integration
+published: True
+position: 10
+---
+
+# Telerik REPL for Blazor Integration
+
+Telerik REPL for Blazor is integrated in our resources (documentation and demos), as well as Visual Studio and Visual Studio Code.
+
+In this article:
+
+* [Telerik UI for Blazor documentation Integration](#telerik-ui-for-blazor-documentation-integration)
+* [Telerik UI for Blazor demos Integration](#telerik-ui-for-blazor-demos-integration)
+* [Visual Studio and Visual Studio Code Integration](#visual-studio-and-visual-studio-code-integration)
+
+## Telerik UI for Blazor Documentation Integration
+
+The Telerik UI for Blazor documentation supports integration with the Telerik REPL for Blazor. This provides interactive experience and easy to test snippets for the various components and scenarios. Above the code snippets listed in the documentation articles you will find a toolbar with the following options:
```

Comment:
The first sentence is too generic and doesn't bring value for the reader. Instead, we can directly start with what the integration benefit is.

Use straight-forward word order - "[Subject] [where] [verb] [info]" is better than "[Where] you will find [subject] [info]"

```suggestion
Most code examples in the Telerik UI for Blazor documentation are directly runnable in Telerik REPL for Blazor. This provides interactive experience, as the snippets are easy to test and edit. The toolbar above each runnable code snippet contains the following options:
```